spec/lib/block_importer_spec.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+describe "#import blocks" do
```

Comment:
The `block_importer_spec.rb` should also be in the `spec/lib/archive_importer` directory (as is the `block_importer.rb` is also in a `archive_importer` sub-directory), and then it should be wrapped in `describe ArchiveImporter::BlockImporter do`, and then only test the stuff from the `BlockImporter`.

So it doesn't create an instance of `ArchiveImporter`, it only creates instances of `BlockImporter` (and run `.import` on it) and then test the different scenarios you want to test for it.

You can have a look at the other tests in `spec/lib/archive_importer`, but since the `BlockImporter` is a simpler than other importers (it's basically just a single diaspora ID as string, and not a complex json structure as parameter), you tests probably also going to look simpler.

(also you still have to remove the modifications from the `archive_importer_spec.rb`)

Reply:
> (also you still have to remove the modifications from the `archive_importer_spec.rb`)

Removed that myself.